scripts/commit_checker.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -ex
+#
+# Copyright (C) 2020 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+#
+
+COMMIT_MESSAGE="`git log -3`"
```

Comment:
Why -3? These are the last 3 commits.
Actually Travis provides you this already in the env variables -> https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

Actually you have multiple variables that help you:
- TRAVIS_COMMIT: The commit that the current build is testing.
- TRAVIS_COMMIT_MESSAGE: The commit subject and body, unwrapped.
- TRAVIS_COMMIT_RANGE: The range of commits that were included in the push or pull request. (Note that this is empty for builds triggered by the initial commit of a new branch.)

Also `git log --pretty=format:'%B' -1` gives you only the body of the commit message.